src/main/java/edu/mbrooks/advancedjava/impl/Person.java

```diff
@@ -0,0 +1,107 @@
+/**
+ * Program Name:    Person.java
+ * Author:          Michael Brooks
+ * Version:         Version 1.0
+ * Course:          Advanced Java Programming, Summer 2018
+ * Copyright:       (C) 2018 Michael A. Brooks
+ * Description:     Implementation of the person interface
```

Comment:
It is ok to have a file header with meta data like copyright but the description should go below in the class JavaDoc documentation.  Also, the file header doc should be not be JavaDoc